Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -163,6 +163,12 @@ bool CBoot::RunApploader(bool is_wii, const DiscIO::VolumeDisc& volume)
   // return
   PC = PowerPC::ppcState.gpr[3];
 
+  if (volume.GetVolumeType() == DiscIO::Platform::Triforce)
+  {
+    char* ptr = (char*)(Memory::GetPointer(0));
+    memset(ptr, 0, 12);
```

Comment:
We have `Memory::Memset`, maybe you could use that instead?

Also: is there any struct we have that corresponds to the 12 bytes being null'ed, or can we at least name that magic number?

Reply:
I wouldn't call it a magic number per se, this is blanking the session key for the encryption of triforce DI commands